app/src/main/res/layout/activity_splash_screen.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Are you sure you wanted to upload this file?

Reply:
no, that's a mistake